src/kademlia/dht_tracker.cpp

```diff
@@ -197,7 +197,7 @@ namespace libtorrent { namespace dht
 		m_blocker.set_rate_limit(m_settings.block_ratelimit);
 
 		error_code ec;
-		m_refresh_timer.expires_from_now(seconds(5), ec);
+		m_refresh_timer.expires_from_now(seconds(m_settings.refresh_timeout), ec);
```

Comment:
if so, this would be calculated by dividing the timeout by the number of nodes in the routing table (probably with some lower bound, like 50-100 nodes). It's important that the actual timeout doesn't get too long, because it's likely that the number of nodes will increase while we're waiting